03 Misc/08 DefinePlugin advanced/prod.webpack.config.js

```diff
@@ -68,4 +69,4 @@ module.exports = function () {
       })
     ],
   });
-}
+})();
```

Comment:
It was a mistake. Could you please update sample with this change to resolve this PR? Thank you so much for you contribution